spec/models/miq_worker_type_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe MiqWorkerType do
+  it ".seed doesn't create a row for excluded classes" do
+    described_class.seed
+    expect(described_class.count).to be > 0
+    expect(described_class.pluck(:worker_type)).not_to include(described_class::EXCLUDED_CLASSES.map(&:to_s))
+  end
```

Comment:
I think we want at least one .seed test showing a known worker type (that is unlikely to go away) added to the table.  Technically the test above will still pass if there are 0 records in the database.

Reply:
That one expectation will pass, but that's also why I added the `expect(described_class.count).to be > 0`.

But yes, I can add a test for generic and ui or something like that.

Reply:
Yeah was just going to say I saw the count...still think a more explicit test might be better though :)

Mark as resolved anyway.

Reply:
Done